numpy/_core/src/multiarray/descriptor.c

```diff
@@ -2028,11 +2028,15 @@ static PyMemberDef arraydescr_members[] = {
     {"byteorder",
         T_CHAR, offsetof(PyArray_Descr, byteorder), READONLY, NULL},
     {"itemsize",
-        T_INT, offsetof(PyArray_Descr, elsize), READONLY, NULL},
+        T_PYSSIZET, offsetof(PyArray_Descr, elsize), READONLY, NULL},
     {"alignment",
-        T_INT, offsetof(PyArray_Descr, alignment), READONLY, NULL},
+        T_PYSSIZET, offsetof(PyArray_Descr, alignment), READONLY, NULL},
     {"flags",
-        T_BYTE, offsetof(PyArray_Descr, flags), READONLY, NULL},
+#if NPY_LONGLONG == NPY_INT64
+        T_LONGLONG, offsetof(PyArray_Descr, flags), READONLY, NULL},
+#else
+  #error Assuming long long is 64bit
```

Comment:
Why is this needed?

Reply:
Because I typd it as `uint64`, but of course you can argue that this is obvious and probably checked elsewhere :)